frame/bags-list/src/list/mod.rs

```diff
@@ -443,8 +443,7 @@ impl<T: Config<I>, I: 'static> List<T, I> {
 
 		// remove the heavier node from this list. Note that this removes the node from storage and
 		// decrements the node counter.
-		// defensive: both nodes have been checked to exist.
-		let _ = Self::remove(&heavier_id).defensive();
+		let _ = Self::remove(&heavier_id).defensive("both nodes have been checked to exist; qed");
```

Comment:
Alternatively could we do?  
```suggestion
		let _ = Self::remove(&heavier_id).with_proof("both nodes have been checked to exist; qed").defensive();
```
```suggestion
		let _ = Self::remove(&heavier_id).defensive("both nodes have been checked to exist; qed");
```

Reply:
sorry slow internet